workers/unity/Assets/Playground/Scripts/Metrics/MetricSendSystem.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections.Generic;
+using Improbable.Gdk.Core.Config;
+using Unity.Entities;
+using UnityEngine;
+using Improbable.Gdk.Core;
+
+namespace Playground
+{
+    public class MetricSendSystem : ComponentSystem
+    {
+        private WorkerBase worker;
+
+        private float timeElapsedSinceUpdate = 0.0f;
+
+        private List<float> fpsMeasurements = new List<float>();
+
+        protected override void OnCreateManager(int capacity)
+        {
+            base.OnCreateManager(capacity);
+            worker = WorkerRegistry.GetWorkerForWorld(World);
+        }
+
+        protected override void OnUpdate()
+        {
+            if (worker.Connection == null)
+            {
+                return;
+            }
+
+            var connection = worker.Connection;
+
+            timeElapsedSinceUpdate += Time.deltaTime;
+            AddFpsSample();
+            if (timeElapsedSinceUpdate >= MetricConfig.TimeBetweenMetricUpdatesSecs)
+            {
+                timeElapsedSinceUpdate = 0;
+                float fps = CalculateFps();
+                var load = MetricConfig.CalculateLoad == null ?
+                    DefaultLoadCalculation(fps) : MetricConfig.CalculateLoad(fps);
+                Worker.Metrics metrics = new Worker.Metrics
+                {
+                    Load = load
+                };
+                connection.SendMetrics(metrics);
+            }
+        }
+
+        private float DefaultLoadCalculation(float fps)
+        {
+            float targetFps = Application.targetFrameRate;
+            return Mathf.Max(0.0f, (targetFps - fps) / (targetFps - 20.0f));
```

Comment:
Why `- 20.0f`?

Reply:
No more hardcoded value now, it was supposed to represent a value at which fps is too low. Now it's half of targetFps.